<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>785</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inventory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="toolTip">
       <string>First create a new database, click on file-&gt;New database. 
Then follow category list help and object list help instructions</string>
      </property>
      <property name="text">
       <string>First use</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Category list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_helpCat">
          <property name="toolTip">
           <string>Categories help:
Add a new category by clicking on the + button.
Remove the selected category by clicking on the - button.
Edit the selected category by double clicking on a category.
Display a category content by clicking on it.
Perform a research by double click on the search category. (Research save until a new research is performed)</string>
          </property>
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Object list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_mediaHelp">
          <property name="toolTip">
           <string>Object help:
Add a new object by cliking on the + button below
Remove selected object by clicking on the - button below
Edit an object by double clicking on it or by select it and click on the See button</string>
          </property>
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="w_category">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Category list</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="w_media">
          <property name="toolTip">
           <string>Media list</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">Nouvelle colonne</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">Nouvelle colonne</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">Nouvelle colonne</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">Nouvelle colonne</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">Nouvelle colonne</string>
           </property>
          </column>
          <column/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QToolButton" name="w_addCategory">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Add category</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="w_removeCategory">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Remove current category</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="w_lcdCat">
            <property name="toolTip">
             <string>Category count</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;</string>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="w_seeMedia">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>See current media</string>
            </property>
            <property name="text">
             <string>See</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="w_addMedia">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Add media</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="w_removeMedia">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Remove current media</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLCDNumber" name="w_lcdMed">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Media count</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;</string>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNouvelle_utilisateur"/>
    <addaction name="actionSearch"/>
    <addaction name="actionAjout_Utilisateur"/>
    <addaction name="actionSettings"/>
    <addaction name="actionCharger"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionA_propos"/>
    <addaction name="actionA_Propos_Qt"/>
   </widget>
   <widget class="QMenu" name="menuMedia">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionNew_media"/>
    <addaction name="actionRemove"/>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuCategorie">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Category</string>
    </property>
    <addaction name="actionNouvelle_categorie"/>
    <addaction name="actionSupprimer_categorie"/>
    <addaction name="actionEditer_categorie"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuCategorie"/>
   <addaction name="menuMedia"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNouvelle_utilisateur">
   <property name="text">
    <string>&amp;New database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionA_Propos_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionNouvelle_categorie">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionSupprimer_categorie">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionEditer_categorie">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionAjout_Utilisateur">
   <property name="text">
    <string>Set &amp;users</string>
   </property>
  </action>
  <action name="actionNew_media">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
